pyproject.toml

```diff
@@ -44,7 +44,7 @@ dacite = "^1.8.0"
 
 
 [build-system]
-requires = ["poetry-core>=1.9.0"]
+requires = ["poetry-core<2.0"]
```

Comment:
in the future we will need to update these

Reply:
Yes I know, I'll do it when everything is merged!
I'll take inspiration on your work on the current charms to do it, but it was easier for now to keep it <2.0